### ISSUE TITLE
Crash with `Type is not JSON serializable: DateTime`

### ISSUE THREAD

**Issue by leplatrem** [mozilla/jira-bugzilla-integration#106]
Context: https://sentry.io/organizations/mozilla/issues/3416994548/?environment=stage&project=6364263

This happens in ORJson, introduced in #86 

```
  File "src/app/api.py", line 101, in bugzilla_webhook
    return ORJSONResponse(content=result, status_code=200)
  File "starlette/responses.py", line 49, in __init__
    self.body = self.render(content)
  File "fastapi/responses.py", line 34, in render
    return orjson.dumps(content)
```

Probably from a Jira response object.

**Comment by grahamalama** [comment]
This is possibly closed with #112? This might also mean we should improve the data we test with so it matches what real responses look like.

**Comment by leplatrem** [comment]
Yes. 
We haven't seen this error for two weeks, so I guess we can safely close it
